frontend/src/lit-components/pages/profile.js

```diff
@@ -0,0 +1,79 @@
+import { html, LitElement } from 'lit';
+
+import {ProfileStyles } from '../../styles';
+import banner from '../../../public/images/banner-profile.png';
+import profile_picture_default from '../../../public/images/profile-picture-default.jpg';

```

Comment:
See this https://github.com/aaronsh1/food-recipe-suggestions/pull/4#discussion_r875771368 regarding imported images.
Your src tag should just directly point to `/public/images/profile-picture-default.jpg`. Same for the banner.